.gitlab-ci.yml

```diff
@@ -89,6 +91,13 @@ variables:
     - /^pre-v[0-9]+\.[0-9]+-[0-9a-f]+$/
     - web
 
+# .chaos-only:                       &chaos-only
```

Comment:
What are these lines for? They are not picked up by Gitlab, right? If so, can they be removed?